airtime_mvc/build/airtime-setup/load.php

```diff
@@ -154,7 +154,7 @@ function checkLiquidsoapService() {
 function checkCeleryService() {
     exec("pgrep -f -u celery airtime-celery", $out, $status);
     if (array_key_exists(0, $out) && $status == 0) {
-        return posix_kill(rtrim($out[0]), 0);
+        return 1;
```

Comment:
Shouldn't this be `return true`? It looks like the `return $status == 0` below is returning a bool and the `1` here is likely getting coerced into a bool by the caller.

Reply:
Yeah it should probably be true because the 1 is just coerced into a bool.